testsuite build warning

```
/home/vagrant/build/tests/gtest-git-prefix/src/gtest-git/googletest/include/gtest/gtest.h: In instantiation of ‘testing::AssertionResult testing::internal::CmpHelperEQ(const char*, const char*, const T1&, const T2&) [with T1 = long unsigned int; T2 = int]’:
/home/vagrant/build/tests/gtest-git-prefix/src/gtest-git/googletest/include/gtest/gtest.h:1554:23:   required from ‘static testing::AssertionResult testing::internal::EqHelper::Compare(const char*, const char*, const T1&, const T2&) [with T1 = long unsigned int; T2 = int; typename std::enable_if<((! std::is_integral<_Tp>::value) || (! std::is_pointer<_Dp>::value))>::type* <anonymous> = 0]’
/vagrant/tests/semantic_analyser.cpp:2275:3:   required from here
/home/vagrant/build/tests/gtest-git-prefix/src/gtest-git/googletest/include/gtest/gtest.h:1527:11: warning: comparison of integer expressions of different signedness: ‘const long unsigned int’ and ‘const int’ [-Wsign-compare]
 1527 |   if (lhs == rhs) {
      |       ~~~~^~~~~~
```

in 

```
TEST(semantic_analyser, tuple_types_unique)
{
  auto bpftrace = get_mock_bpftrace();
  test(*bpftrace, R"_(BEGIN { $t = (1, "str"); $t = (4, "other"); })_", 0);

  EXPECT_EQ(bpftrace->structs.GetTuplesCnt(), 1);
}
```

--- a/tests/semantic_analyser.cpp
+++ b/tests/semantic_analyser.cpp
@@ -2194,7 +2194,7 @@ TEST(semantic_analyser, tuple_types_unique)
   auto bpftrace = get_mock_bpftrace();
   test(*bpftrace, R"_(BEGIN { $t = (1, "str"); $t = (4, "other"); })_", 0);
 
-  EXPECT_EQ(bpftrace->structs.GetTuplesCnt(), 1);
+  EXPECT_EQ(bpftrace->structs.GetTuplesCnt(), 1ul);
 }
 
 TEST(semantic_analyser, multi_pass_type_inference_zero_size_int)

